var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -562,7 +562,7 @@ def define_tpl(trilinos_name, spack_name, have_dep):
             depspec = spec[spack_name]
             libs = depspec.libs
             options.extend([
-                define(trilinos_name + '_INCLUDE_DIRS', depspec.prefix.include),
+                define(trilinos_name + '_INCLUDE_DIRS', depspec.headers.directories[0]),
```

Comment:
I think it should be safe to allow all the directories that show up under `headers.directories`:
```suggestion
                define(trilinos_name + '_INCLUDE_DIRS', depspec.headers.directories),
```
since the `define` macro will convert a Python list into a CMake-style semicolon-separated list, and TriBITS seems to support multiple INCLUDE_DIRS here.

Reply:
I concur - in my testing I initially used your suggestion above.  In practice there is only 1 directory in the list for each of Trilinos' dependencies that make it into this function.  I'll gladly push this mod back in here.  I decided to limit it to a single directory because of what I had to do on lines 634 and 635 (see my comments below).